### PR TITLE
Remove old email alias

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Removing old references to dev-inquiries@okta.com
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* n/a

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/bb0eed2dc98407e9b9cf72dc81b5872180f35c94/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/96986a2d-e3c5-4856-b14d-ab3c1d93476f) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/bb0eed2dc98407e9b9cf72dc81b5872180f35c94/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a424dded-34b7-479d-8e59-08f523284bda) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/bb0eed2dc98407e9b9cf72dc81b5872180f35c94/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 511 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3792bbcc-8406-4d72-b479-54f15d6814bd) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.8a2281ef-eb24-4151-9c32-d3053b3cd5eb?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Explore the [Okta Public API Collections](https://www.postman.com/okta-eng/works
 
 <ApiLifecycle access="ie" />
 
-> **Note:** This feature is only available as a part of Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+> **Note:** This feature is only available as a part of Okta Identity Engine. Please [contact support](https://support.okta.com/) for further information.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:16]:
```suggestion
> **Note:** This feature is only available as a part of Okta Identity Engine. [Contact support](https://support.okta.com/)  for information on the Identity Engine.
```

File: packages/@okta/vuepress-site/docs/reference/api/devices/index.md [hunk 2]
```diff
@@ -727,7 +727,7 @@ curl -v -X GET \
 
 Permanently deletes a Device that is in `DEACTIVATED` status. The Device can be transitioned to `DEACTIVATED` status using the [deactivate](#deactivate-device) API.
 
-This deletion is destructive and deletes all the profile data related to the device. Once deleted, device data can't be recovered. A Device that isn’t in a `DEACTIVATED` state raises an error if a delete operation is attempted.
+This deletion is destructive and deletes all the profile data related to the device. Once deleted, device data can't be recovered. A Device that isn't in a `DEACTIVATED` state raises an error if a delete operation is attempted.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/devices/index.md:730]:
```suggestion
This deletion is destructive and deletes all the profile data related to the device. After it's deleted, device data can't be recovered. A Device that isn't in a `DEACTIVATED` state raises an error if a delete operation is attempted.
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 3]
```diff
@@ -211,7 +211,7 @@ HTTP 204:
 
 <ApiLifecycle access="ie" />
 
-> **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+> **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](https://support.okta.com/) for further information.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:214]:
```suggestion
> **Note:** This feature is only available as a part of the Identity Engine. [Contact support](https://support.okta.com/) for information on the Identity Engine.
```

File: packages/@okta/vuepress-site/docs/reference/api/captchas/index.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ Explore the [Okta Public API Collections](https://www.postman.com/okta-eng/works
 
 <ApiLifecycle access="ie" />
 
-> **Note:** This feature is only available as a part of Okta Identity Engine. [Contact support](mailto:dev-inquiries@okta.com) for further information.
+> **Note:** This feature is only available as a part of Okta Identity Engine. [Contact support](https://support.okta.com/) for further information.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/captchas/index.md:16]:
```suggestion
> **Note:** This feature is only available as a part of Okta Identity Engine. [Contact support](https://support.okta.com/) for information on the Identity Engine. 
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 5]
```diff
@@ -1280,7 +1280,7 @@ Specifies a particular platform or device to match on
...
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:1283]:
```suggestion
> **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](https://support.okta.com/) for information on the Identity Engine.
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 6]
```diff
@@ -2121,7 +2121,7 @@ You can apply the following conditions to the IdP Discovery Policy:
 | providers | List of configured Identity Providers that a given Rule can route to | array     | Yes      |
 
 > **Note:** Ability to define multiple providers is a part of the Identity Engine.
-> Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+> Please [contact support](https://support.okta.com/) for further information.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:2124]:
```suggestion
> [Contact support](https://support.okta.com/) for information on the Identity Engine.
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 7]
```diff
@@ -2144,7 +2144,7 @@ You can apply the following conditions to the IdP Discovery Policy:
 
 <ApiLifecycle access="ie" />
 
-> **Note:** This feature is only available as a part of the Identity Engine. [Contact support](mailto:dev-inquiries@okta.com) for further information.
+> **Note:** This feature is only available as a part of the Identity Engine. [Contact support](https://support.okta.com/) for further information.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:2147]:
```suggestion
> **Note:** This feature is only available as a part of the Identity Engine. [Contact support](https://support.okta.com/) for information on the Identity Engine.
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 8]
```diff
@@ -2224,7 +2224,7 @@ refers to the user's `username`. If the user is signing in with the username `jo
...
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:2227]:
```suggestion
> **Note:** This feature is only available as a part of the Identity Engine. [Contact support](https://support.okta.com/) for information on the Identity Engine.
```

**Review by susanharper-okta — DISMISSED** [review]
some style issues with some fairly old docs, I'm assuming. I'll have to approve again, now. So just Slack me and I'll approve.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/825c3f76323513c82c2f4aa1e414fccbd2ade90c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bf552d99-9450-4fc5-81d0-4567139c6964) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/825c3f76323513c82c2f4aa1e414fccbd2ade90c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/03158520-6f11-469d-b3cf-dad039608cb3) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/825c3f76323513c82c2f4aa1e414fccbd2ade90c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 511 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0c4146e2-5e41-446c-8867-fa4a27237fc4) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.de12181d-49e3-4ebb-97d7-803a7f6b0c26?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/5dc39155a4324bd79cd6393dba3a1b1f5edfb066/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4c2b2c5d-aa57-40d9-8356-1e36bfb76a6b) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/5dc39155a4324bd79cd6393dba3a1b1f5edfb066/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/be7f91d1-4a11-4905-a9e5-eada936b9ac3) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/5dc39155a4324bd79cd6393dba3a1b1f5edfb066/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 511 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f4b9f2d0-bc1e-4043-a066-a8efeb2573bf) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.f4feee85-f53d-4b67-be2f-0b410000235f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/6e4ecbe3487a16d04cec39d8d3aa2a91398ccde0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/862d079b-da31-4f83-b7bc-4397999f3290) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/6e4ecbe3487a16d04cec39d8d3aa2a91398ccde0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e26e6865-9e4f-4892-be7f-c5df7af15f7f) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/6e4ecbe3487a16d04cec39d8d3aa2a91398ccde0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 510 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5fb66085-989d-40fc-943a-a0cdabf5e55b) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.56b95733-7f1b-4f7f-9c19-92ef31eef079?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/34b16d55-a310-43b7-a957-552513f4363e) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a4c0db44-688a-4703-a79a-157f0307f08b) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 509 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7a9d7e62-a41f-462a-b9a4-c62b03f2d73b) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.257afee6-241c-429e-b685-be9d4252e335?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1ba218b5-9d5c-4717-a9b7-b799e9c842e4) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bca482c0-dae6-4741-adfb-4bbdb3b96b91) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 509 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/89df790f-36e4-4275-a6bc-5a33b183862a) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.40952f98-aa62-435f-ac5e-e0a7178748b3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/cf36616e-d6c7-46cc-8272-481c3285a84a) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/798ca0f6-7dbc-45f5-bb30-7969920b3a19) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 509 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d39674a3-8029-4640-a4a8-45bb7bb00ab5) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.a2ede68f-faa9-4350-bc85-ac654b7f342f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c9d2fa68-4df2-401b-a41f-1f464590d827) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dad70c02-b0cd-4bd1-8aa6-994f24b49a65) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 509 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/559c91c5-ae89-45a5-a012-be79eae04f63) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.c04bc8b5-66f4-4c49-8949-4979629583be?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8ec2d101-1105-4532-9cd4-ca5984b9bb5d) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fd0159de-ea3e-49b9-8b5a-98b36a522d91) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/4fe5087f7f6ec90888e180c92705778960aeac17/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 509 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f737537c-f514-4211-900e-651b5ac48625) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.7963db61-f6f0-40fe-834e-2accb2c0d867?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by susanharper-okta — COMMENTED** [review]
couple comments on the note that was added

File: packages/@okta/vuepress-site/pricing/index.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
-# This file controls the links and content shown on the /pricing landing page.
+# This file controls the links and content shown on the /pricing landing page. NOTE: as of Jan 2024, this page no longer builds the pricing page, which is a redirect to: https://www.okta.com/pricing/#customer-identity-products
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/pricing/index.md:2]:
```suggestion
# This file controls the links and content shown on the /pricing landing page. **Note:** As of Jan 2024, this page no longer builds the pricing page, which is a redirect to `https://www.okta.com/pricing/#customer-identity-products`.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/7eea1bc5dea6c87b7eee08976fdd690b23045812/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1712af5c-c322-462f-9861-65fb54508f7c) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/7eea1bc5dea6c87b7eee08976fdd690b23045812/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c850c6cb-17a7-4038-bb17-c9dd893bdfb4) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/7eea1bc5dea6c87b7eee08976fdd690b23045812/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 509 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b161620d-c808-4f3d-8d97-c6416dbfe943) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/pricing/index.md](https://github.com/okta/okta-developer-docs/blob/7eea1bc5dea6c87b7eee08976fdd690b23045812/packages%2F%40okta%2Fvuepress-site%2Fpricing%2Findex.md) | 89 | 7 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/258a87ef-6174-4c9d-be5d-d9781ac9b559) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.be7e1f65-568f-49ce-82c1-337f97d50c96?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/captchas/index.md](https://github.com/okta/okta-developer-docs/blob/2611bb72924f40955f94061bdda2d10f2ae15a91/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fcaptchas%2Findex.md) | 85 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/59260f09-2010-4e44-871b-2fc6e2215bae) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/devices/index.md](https://github.com/okta/okta-developer-docs/blob/2611bb72924f40955f94061bdda2d10f2ae15a91/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fdevices%2Findex.md) | 73 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d63026a0-19a1-4d52-a8a9-d613d1569860) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/2611bb72924f40955f94061bdda2d10f2ae15a91/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 509 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/af943565-b580-4c64-ad65-702cba3a3a54) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/pricing/index.md](https://github.com/okta/okta-developer-docs/blob/2611bb72924f40955f94061bdda2d10f2ae15a91/packages%2F%40okta%2Fvuepress-site%2Fpricing%2Findex.md) | 89 | 7 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/119e417a-0169-427a-98a4-64b8d4422c98) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 4 of 4 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.c8c6e7f3-3823-49f7-8ea3-2957ea0afe70?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**